packages/playground/src/calculator/resource_pricing.vue

```diff
@@ -209,6 +211,9 @@ const isCertified = ref<boolean>(false);
 const ipv4 = ref<boolean>(false);
 const currentbalance = ref<boolean>(false);
 
+const profileManagerController = useProfileManagerController();
+const freeBalance = computed(() => profileManagerController.balance.value?.free ?? 0);
```

Comment:
i suggest to set it to fixed 2 decimals 

```suggestion
const freeBalance = computed(() => (profileManagerController.balance.value?.free)?.toFixed(2) ?? 0);
```

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/62248851/0df77b43-587d-4dd4-943d-3e8b357a51c5)